webrtc.html

```diff
@@ -9375,7 +9375,8 @@ <h4>Closing procedure</h4>
         <li>
           <p>Unless the procedure was initiated by <var>channel</var>.{{RTCDataChannel/close}}, set
           <var>channel</var>.<a>[[\ReadyState]]</a> to
-          {{RTCDataChannelState/"closing"}}.</p>
+          {{RTCDataChannelState/"closing"}} and <a>fire an event</a>
+          named "closing" at <var>channel</var></p>
```

Comment:
Missing period.

Also, for consistency, we should use same formatting (s/"closing"/`closing`/) as elsewhere.